src/sentry/testutils/factories.py

```diff
@@ -1113,6 +1151,7 @@ def create_doc_integration(features=None, has_avatar: bool = False, **kwargs) ->
 
     @staticmethod
     @exempt_from_silo_limits()
+    @outbox_runner()
```

Comment:
Is this something we could use as a method annotation in application code? It could be useful in that context as well.

Reply:
outbox_runner by default runs *all* outboxes it finds -- that's why it is useful in tests.  That may be heavy for application code.

Reply:
That's fair, we don't have many scenarios where we would want that.